README.md

```diff
@@ -29,6 +29,16 @@ cd reviewcheck
 poetry run reviewcheck
 ```
 
+### Docker
```

Comment:
I think we should add headings for the other two ways also then. Maybe "With pip" and "Using poetry"?

Reply:
fixed